packages/react/test/client-generator.test.ts

```diff
@@ -37,7 +37,9 @@ const router = {
     }),
 };
 
-const { useUploadThing } = generateReactHelpers<typeof router>();
+const { useUploadThing } = generateReactHelpers<typeof router>({
+  url: "0.0.0.0",
```

Comment:
Would this be a valid url?

Reply:
probably not - but the test isn't executing any requests so doesn't really matter?